Calculator/index2.html

```diff
@@ -0,0 +1,16 @@
+<! DOCTYPE HTML>
```

Comment:
rename the file to `index.html` also your code style is wrong.
http://codeguide.co/